### ISSUE TITLE
Split terminal or delete terminal should keep focus on the tab list

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#122334]
Testing #122242

Keyboard focus on to the terminal list. Using keyboard delete or split a terminal. Focus is now lost.
For inspiration checkout how the explorer focuses the neighbour item when an element gets deleted. I would expect the terminal tab list to also do the same thus keeping focus stable

**Comment by Tyriar** [comment]
Think I closed this off incorrectly.

**Comment by meganrogge** [comment]
Related, I cannot currently delete a terminal via the keyboard. 
![recording - 2021-04-27T135716 952](https://user-images.githubusercontent.com/29464607/116311764-78c7ef80-a760-11eb-96ec-d6a803b97eff.gif)

**Comment by isidorn** [comment]
Delete terminal nicely keeps focus, however split terminal does not.
Steps:

1. Focus into terminal tab list
2. Using keyboard focus onto the split terminal action
3. Space -> terminal is split, focus is lost 🐛

**Comment by isidorn** [comment]
This works great now -> verified
